src/components/gas/GasSpeedButton.js

```diff
@@ -323,19 +323,30 @@ const GasSpeedButton = ({
   }, [currentNetwork, speeds]);
 
   const menuConfig = useMemo(() => {
-    const menuOptions = speedOptions.map(gasOption => ({
-      actionKey: gasOption,
-      actionTitle: upperFirst(gasOption),
-      icon: {
-        iconType: 'ASSET',
-        iconValue: GAS_ICONS[gasOption],
-      },
-    }));
+    const menuOptions = speedOptions.map(gasOption => {
+      const totalGwei = add(
+        gasFeeParamsBySpeed[gasOption]?.maxFeePerGas?.gwei,
+        gasFeeParamsBySpeed[gasOption]?.maxPriorityFeePerGas?.gwei
+      );
+      const gweiDisplay =
+        currentNetwork === networkTypes.polygon
+          ? gasFeeParamsBySpeed[gasOption]?.gasPrice?.display
+          : `${parseInt(totalGwei, 10)} Gwei`;
```

Comment:
`parseInt` can be a bit misleading (e.g, say a user has 4.9, we would show 4 gwei instead of 5)

instead we've been using:
`toFixedDecimals(totalGwei, 0)` which does "half up rounding" and returns a string from `utilities`

Reply:
done 👌